lighthouse-cli/test/fixtures/form.html

```diff
@@ -0,0 +1,122 @@
+<html><head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Autofill Smoke Test</title>
+</head>
+<body>
+  <form id="checkout" name="checkout" action="../done.html" method="post">
+      <label id="cardName" for="name_cc">Name on card:</label> <textarea type="text" name="name_cc" autocomplete="cc-name"></textarea><br>
```

Comment:
we also try to annotate the elements that appear so we know what test they're supposed to apply to, we don't have an audit yet, so for now maybe just an explanation of why it's here?

```suggestion
      <!-- FormElements Gatherer - textarea example -->
      <label id="cardName" for="name_cc">Name on card:</label> <textarea type="text" name="name_cc" autocomplete="cc-name"></textarea><br>
```